src/garage/tf/plotter/plotter.py

```diff
@@ -53,11 +53,11 @@ def __init__(self,
                  rollout=default_rollout):
         Plotter.__plotters.append(self)
         self._env = env
-        self._policy = policy
         self.sess = tf.compat.v1.Session() if sess is None else sess
-
         self.graph = tf.compat.v1.get_default_graph(
         ) if graph is None else graph
+        with self.sess.as_default(), self.sess.graph.as_default():
+            self._policy = policy.clone('plotter_policy')
```

Comment:
Can this be done inside `LocalRunner`?

Reply:
Not really, this policy has to be cloned and built (automatically during construction) under the session which the plotter would use, so it needs to be done here.

Reply:
Unless we pass the current default tf.Session, then we can also pass a "cloned" policy.

Reply:
Using default tf session will break the plotter (not this tf one), so maybe not.